pangeo-binder/requirements.yaml

```diff
@@ -1,14 +1,15 @@
 # requirements.yaml
+# https://github.com/jupyterhub/mybinder.org-deploy/blob/a15bac97e26b8a085255e06b3c765b9fb2e982fd/mybinder/Chart.yaml
 dependencies:
 - name: binderhub
   version: 0.2.0-n219.hbc17443
```

Comment:
this is pretty far behind mybinder.org, but i don't want to mess with it since things are working